test/commonjs_bridge.spec.js

```diff
@@ -11,7 +11,7 @@ describe('client', function() {
     describe('resolve from a file', function () {
 
       beforeEach(function () {
-        window.__cjs_module__['/folder/foo.js'] = function(require, module, exports) {
+        window.__cjs_module__['/folder/foo.js'] = function(require, module, exports, global) {
```

Comment:
You don't need to add this argument if it is not used in a test. Same applies to other changes like this.